packages/infra/infra-shared/src/stacks/ci/ciBackend.ts

```diff
@@ -107,6 +108,15 @@ export class BackendCiConfig extends ServiceCiConfig {
       },
       cache: codebuild.Cache.local(codebuild.LocalCacheMode.DOCKER_LAYER),
     });
+
+    project.addToRolePolicy(
```

Comment:
This way backend stack will have access to every KMS keys in the account. WDYT about giving access to the key from bootstrap stack?

Reply:
Yeah, you are right